usr/lib/cca_stdll/ccatok.conf

```diff
@@ -31,6 +31,10 @@ version cca-0
 # IBM specific boolean attribute CKA_IBM_PROTKEY_EXTRACTABLE must be true to
 # make a key eligible for protected key support. 
 #
+# AES-XTS related mechanisms are only available if the PKEY_MODE option is 
+# enabled and additional hardware and firmware prerequisites are met. AES-XTS
```

Comment:
should we use upper case `ENABLED` here, as the option value is `ENABLED` ?

Reply:
Hmm, good question ... I intentionally used "enabled" here to make it general. So we can use the same text also for ep11 where we have different forms of "enabled". OK, we may use "not disabled" as you suggest below for ep11.

Reply:
Yes using 'not disabled' for both is probably the best, and is still general.

Reply:
ok, text changed.